docs/README.md

```diff
@@ -0,0 +1,69 @@
+# Kite for Visual Studio Code
+
+This extensions adds support for Kite in Visual Studio Code, including:
+
+- Completion Lists
+- Signature Help
+- Quick Info
+- Goto Definition 
+
+## How to use this extension
+
+This extension is automatically installed by Kite when you select Visual Studio Code in the list of plugins.
+
+### Startup
+
+When starting VSCode with Kite's plugin for the first time, the Kite's tour will be displayed in the active pane.
```

Comment:
Change:
* `Kite's plugin` → `Kite`
* `the Kite's tour` → `a brief tour about Kite`